### DISCUSSION TITLE
How to Change Slug of URL

### DISCUSSION THREAD

**Discussion by ElenaCF** [vercel/next.js#66256] [Help]
### Summary

Hello,

I am new to modifying website templates on Vercel. I am trying to change the slug of "[/work](https://webdev-portfolio-smoky.vercel.app/work)" to "/about", and "[/blog](https://webdev-portfolio-smoky.vercel.app/blog)" to "/portfolio" and have everything work as it did before, but with new slugs.

How should I go about doing that?

### Example

https://webdev-portfolio-smoky.vercel.app/

**Comment by versecafe** [comment]
@ElenaCF You can just rename the folders to change the slug so changing a folder from /work to /about changes the slug the same way
